scripts/partition_manager/partition_manager.rst

```diff
@@ -223,7 +287,7 @@ Configuration file preprocessing
 
 Each :file:`pm.yml` file is preprocessed to resolve symbols from Kconfig and DTS.
 
-The following example shows a typical :file:`pm.yml` file.
+The following example shows a :file:`pm.yml` file for a stage zero bootloader partition named ``b0``.
```

Comment:
```suggestion
The following example shows a :file:`pm.yml` file for an immutable bootloader partition named ``b0``.
```

Reply:
How about:

```
The following example is taken from the :file:`pm.yml` file for an immutable bootloader application.
```

Looking at this again, I see that the fact that the partition's name is `b0` is already documented in a YAML comment.

Reply:
You could actually link to the immutable bootloader then: :ref:`bootloader`